lib/jekyll/scholar/tags/bibliography_count.rb

```diff
@@ -0,0 +1,53 @@
+module Jekyll
+  class Scholar
+
+    class BibliographyCountTag < Liquid::Tag
+      include Scholar::Utilities
+
+      def initialize(tag_name, arguments, tokens)
+        super
+
+        @config = Scholar.defaults.dup
+
+        optparse(arguments)
+      end
+
+      def render(context)
+        set_context_to context
+
+        # Add bibtex files to dependency tree
+        if context.registers[:page] and context.registers[:page].key? "path"
+          bibtex_paths.each do |bibtex_path|
+            site.regenerator.add_dependency(
+              site.in_source_dir(context.registers[:page]["path"]),
+              bibtex_path
+            )
+          end
+        end
+
+        items = entries
+
+        if cited_only?
```

Comment:
I think there is something wrong here? We return `entries.size` at the end, so this whole block adjusting `items` has no effect. We probably want to return `items.size`? In that case, we should, again, move this to `utilities` so that `bibliography` and `bibliography_count` can share it.